react/README.md

```diff
@@ -282,7 +286,7 @@ export default class TextInput extends React.Component {
 
 **[⬆ back to top](#table-of-contents)**
 
-### Ordering
+### `propTypes` Ordering
```

Comment:
I know it's not part of this commit, but was the extra line on line after `onChange propType` #299 intentional?